src/App.vue

```diff
@@ -17,6 +17,7 @@
         :images="images" 
         ref="lightbox"
         :show-caption="true"
+        :closable="false"
```

Comment:
Can you set it to `true`. I think the demo app should be demonstrate normal use cases

Reply:
Yes I forgot to change it back! thanks